service/service_construction.go

```diff
@@ -23,9 +23,12 @@ import (
 
 const (
 	genericTransferBytesLength = 68
+	genericUnwrapBytesLength   = 68
```

Comment:
Is this referenced somewhere? May be good practice to indicate how these are computed.

Reply:
Added comment to explain these:

// 68 Bytes = methodID (4 Bytes) + param 1 (32 Bytes) + param 2 (32 Bytes)